.editorconfig

```diff
@@ -9,5 +9,5 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[*.java]
+[*.{java,xml}]
```

Comment:
why?

Reply:
- same as other projects
- xml default indentation is 4 spaces

used terminal `cp` command from one of the recent projects we upgraded and it did not use auto-indentation. scanned the file - no changes. but 4 spaces is default so i'd rather keep that